### ISSUE TITLE
Add MeshCentral console to "Run Commands" (group) actions

### ISSUE THREAD

**Issue by MailYouLater** [Ylianst/MeshCentral#4343]
I'd like to be able to run console commands (as in commands that would normally be run in a device's Console tab) on multiple devices at once via the "Group Action" functionality.  Currently the Run Command option in the "Group Action" dialog has options to run three kinds of commands: "Windows Command Prompt", "Windows PowerShell", and "Linux/BSD/macOS Command Shell".  I'd like to see MeshCentral's console added to that list so that I can select several devices, click "Group Action" and run things like the `agentupdate` or `alert` commands.

Basically, I'd like to open this dialog:
![image](https://user-images.githubusercontent.com/1279475/181842506-5eb70322-52f4-4e7d-b88e-bd44358fe561.png)
And run the commands I currently run on this screen:
![image](https://user-images.githubusercontent.com/1279475/181863851-1c62d7fe-5232-40f4-ac57-593851b8924a.png)

**Comment by silversword411** [comment]
Console is more for agent diagnostics. 

Agent updates happen automatically, why are you doing it manually?

**Comment by Ylianst** [comment]
This is not a bad idea and not too difficult to implement. I am going to do it since it may be useful for more own testing of the agent.

**Comment by Ylianst** [comment]
Check it in and looks like it works. Will be in v1.0.61.

![MC2-BatchAgentConsoleCommands](https://user-images.githubusercontent.com/1319013/181949348-5429761f-db75-42dd-b940-11e7ca62413c.png)

**Comment by MailYouLater** [comment]
> Agent updates happen automatically, why are you doing it manually?

Sometimes I want to change options inside an agent's `.msh` file and reboot the agent to ensure the changes have taken effect.
I've found that going to the Console tab and typing `updateagent` is easer & quicker than going to the Desktop tab, clicking the Tools button, waiting for the Services tab to appear, clicking the services tab, scrolling down to find "Mesh Agent", clicking on it, and then clicking "Restart".  I also sometimes want to do this on several computers at once.  There are also several other agent console commands which I occasionally want to run and would be nice to be able to run them on multiple devices at once.

> Check it in and looks like it works. Will be in v1.0.61.

Thank you.
